README.rdoc

```diff
@@ -1,6 +1,6 @@
 == QATest
 
-Проектом QATest предоставляется HTTP API по работе со списками пользователей. Результат команд выдается в json-формате:
+Проектом QATest предоставляется HTTP API по работе со списками пользователей. Результат команд выдается в JSON-формате:
```

Comment:
I haven't changed a lot here, because:

1. Not sure that the LICENSE file is required for this repo.
2. As I got it, RDoc is more suitable for the repo rather than Markdown.
3. Not sure that “Sponsored by FunBox” is the right thing for this repo.

So, there're just small text changes.